articles/azure-app-configuration/concept-enable-rbac.md

```diff
@@ -33,5 +33,8 @@ Azure provides the following Azure built-in roles for authorizing access to App
 - **Contributor**: Use this role to manage the App Configuration resource. While the App Configuration data can be accessed using access keys, this role does not grant direct access to the data using Azure AD.
 - **Reader**: Use this role to give read access to the App Configuration resource. This does not grant access to the resource's access keys, nor to the data stored in App Configuration.
 
+> [!NOTE]
+> After role assignment is made for an identity, allow 15 minutes for the permission to propagate before accessing data stored in App Configuration using the identity.
```

Comment:
Small catch: after **a** role assignment....

and then at the end "App Configuration using **this** identity"

Reply:
Thanks. Updated.